components/infobox/extensions/wikis/rocketleague/years_active.lua

```diff
@@ -7,20 +7,86 @@
 --
 
 local Class = require('Module:Class')
+local Lpdb = require('Module:Lpdb')
 local Lua = require('Module:Lua')
+local Set = require('Module:Set')
+local Table = require('Module:Table')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
 
 local CustomActiveYears = Lua.import('Module:YearsActive/Base', {requireDevIfEnabled = true})
 
 -- wiki specific settings
 CustomActiveYears.defaultNumberOfStoredPlayersPerPlacement = 6
 CustomActiveYears.additionalConditions = ''
 
+local _TALENT_POSITIONS = require('Module:BroadcastPositions').TalentPositions
```

Comment:
Please push the data module to git as well (see #1410)

Reply:
Doesn't seem to make sense given the conversation we had about it

Reply:
It's needed to keep the dev wikis working cleanly
Updating a specific data module is simpler for most users than when it's somewhere in the code
Put it in the same folder